src/main/index.js

```diff
@@ -21,8 +22,19 @@ function runApp() {
     showSaveImageAs: true,
     showCopyImageAddress: true,
     prepend: (defaultActions, parameters, browserWindow) => [
+      // TODO: Find a way to move "paste and go" below the default "paste" action.
       {
-        label: 'Show Video Statistics',
+        label: 'Paste and Go',
+        visible: parameters.isEditable,
+        enabled: clipboard.readText(),
+        click() {
+          const target = browserWindow.webContents || (browserWindow.id && browserWindow)
+          target.paste()
+          target.sendInputEvent({ type: 'keydown', keyCode: 'Enter' })
+        }
+      },
+      {
+        label: 'Toggle Video Statistics',
```

Comment:
change to [Show / Hide Video Statistics](https://github.com/FreeTubeApp/FreeTube/issues/2277#issuecomment-1146831705)